app/views/servers/check_installation.js.erb

```diff
@@ -2,5 +2,20 @@
   $("[data-server-status~=checking]").addClass("hidden");
   $("[data-server-status~=connected]").removeClass("hidden");
 <% else %>
+  <% if @server.install_step == 1 %>
```

Comment:
I was thinking about that, but I want to improve this even further by adding more steps. When this becomes an enum you kinda lock yourself in. At this point a step is called: `Installing Docker`, but there is a big change it will change in the future